storage/README.md

```diff
@@ -1,9 +1,26 @@
 # Storage layer
 
-The interface, various concrete implementations, and any associated components live here.
-Currently, there is only one storage implementation:
-   * MySQL/MariaDB, which lives in [mysql/](mysql).
+## Package Layout
 
+The interface, various concrete implementations, and any associated components
+live here. Interfaces and types are defined at the top level package.
+
+Currently, there are two usable storage implementation for logs:
+   * MySQL/MariaDB in the [mysql/](mysql) package.
+   * Cloudspanner in the [cloudspanner](cloudspanner) package.
```

Comment:
"Cloud Spanner" seems to be the official capitalization / spacing. (1 more below).

Reply:
Both fixed.